### PR TITLE
added media query and finish the project

### PR DESCRIPTION
**Pull request by hasnatali1947**
- Added media query
- Added some **div tag** in Html
- Finish all project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,13 +22,13 @@ <h1>My Logo</h1>
 
     <main>
```
Comment (joel-kalema) [inline index.html]: Great work @hasnatali1947 working on your desktop version 😃, but you still need to fix some parts:

- [ ] Kindly make sure you fixed your `nav` 😃 , please remove the icon in desktop version. As we can see the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=noHceN5fZjbUZpNo-0) , the icon is displayed on the mobile version only, but on the desktop version we are displaying the links to your `Portofolio`, `About` and `Contact` 
- [ ] Kind make sure you fixed the `background image` to replicate the design as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.). Also your `icon's margins`, when the `height` of the screen is less than `700px` it's not fixed well. I know you worked with a big screen that is why you can't see it, but if another person with a different screen try to run your project, it gives that error, kindly fix it 👍 
 

### Figma
good

### Your design
desk error

File: index.html [hunk 2]
```diff
@@ -72,246 +72,288 @@ <h3 class="Connect-heading">LET'S CONNECT</h3>
...
         <article class="artical-1">
-          <ul class="portfolio-idcard">
-            <li>
-              <img class="idcard-1-img" src="Portfolio (1).png" alt="" />
-            </li>
-          </ul>
-          <div class="Tonic-1">
-            <a href="#">
-              <h1>Tonic</h1>
-            </a>
```
Comment (joel-kalema) [inline index.html:83]: - [ ] Great work @hasnatali1947 working on your desktop version 😃, but make sure you fixed `margin` to replicate the design as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.). Kindly don't forget to fix you the `style` when we `hover` your `section` and `button` 👍  
### Figma
project good
### Your design
project error

- [ ] Also make sure you fixed the section `width` to to replicate the design as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.).

### Figma
about good
### Your design
about error


- [ ] Please, fix the `background image`  to to replicate the design as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.).

### Figma
contact good
### Your design
contact error

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @hasnatali1947 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -384,5 +428,425 @@ <h2>Contact me</h2>
...
+              </li>
+            </ul>
+          </div>
+          </form>
+        </div>
+      </footer>
+
+  </main>
+  
+</body>
```
Comment (mckent05) [inline index.html:851]: **OPTIONAL** - I see that you have two sections of HTML, one for the mobile version and another for the desktop version, this is not good practice. You are required to have just one part and use CSS (media-queries) to make them responsive for both mobile and desktop. Please I strongly advise you to fix this 🙏

**Review by mckent05 — APPROVED** [review]
Hi @hasnatali1947, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Your portfolio looks good 👍 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**